### ISSUE TITLE
Home Assistant version support

### ISSUE THREAD

**Issue by ruimarinho** [vinteo/hass-opensprinkler#10]
Hi! @vinteo!

Thank you for your work on this module! I wanted to ask you which versions of HASS does this module support (e.g. does it work with 0.80?) and if you've had any experience with the new Lovelace UI yet.

**Comment by vinteo** [comment]
Yes it works on 0.80, I have it running on 0.80.3 right now with Lovelace UI as well
